bot/cogs/trivia.py

```diff
@@ -155,8 +165,9 @@ async def on_message(self, message):
         auth_id = str(message.author.id)
         ctx = message.channel
         user_answer = message.content.lower()
-        try:
-            if userAnswers[auth_id] != 0:
+        if userAnswers[auth_id] != 0:
```

Comment:
auth_id was never in userAnswers, this caused an error. Also: do not use camelCamel case in python